hw/top_earlgrey/ip/rv_plic/data/autogen/BUILD

```diff
@@ -1,14 +0,0 @@
-# Copyright lowRISC contributors.
-# Licensed under the Apache License, Version 2.0, see LICENSE for details.
-# SPDX-License-Identifier: Apache-2.0
-
-package(default_visibility = ["//visibility:public"])
-
-load("//rules:autogen.bzl", "autogen_hjson_header")
-
-autogen_hjson_header(
-    name = "rv_plic_regs",
-    srcs = [
-        "rv_plic.hjson",
-    ],
-)
```

Comment:
it looks like this removes bazel build rules... we probably have to leave these in place (or generate them) since this may otherwise break the new bazel-based SW build flow (CC @drewmacrae to confirm that this is the case).

Reply:
Thanks for flagging this, I have [a PR to fix it](https://github.com/lowRISC/opentitan/pull/8568), if you have bazel installed and you're interested you can test the bazel rules with ```bazel test //...```

I still need to write a getting started doc for it. If you don't have it installed I can point to instructions here: https://docs.bazel.build/versions/main/install-ubuntu.html